cif_pow.dic

```diff
@@ -11967,6 +11967,32 @@ save_pd_proc_ls.diffractogram_id
 
 save_
 
+save_pd_proc_ls.number_parameters
+
+    _definition.id                '_pd_proc_ls.number_parameters'
+    _definition.update            2025-07-20
+    _description.text
+;
+    Number of parameters refined in the least-squares process. If
+    possible this number should include the restrained parameters.
+    The restrained parameters are distinct from the constrained
+    parameters (where one or more parameters are linearly dependent
+    on the refined value of another).  Least-squares restraints
+    often depend on geometry or energy considerations and this
```

Comment:
including restraints "if possible" leads to ambiguity when writing software to read and write data files - does the value actually include restrained parameters or not?  I suggest not, given that it is difficult to assess their contribution. We would describe the restraints elsewhere, and if necessary wait for the theory on how to correctly incorporate restraints into the parameter count.